pubkey_encrypt.install

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the pubkey_encrypt module.
+ */
+
+/**
+ * Implements hook_requirements().
+ */
+function pubkey_encrypt_requirements($phase) {
+  $requirements = array();
+  // Check for requirements during install.
+  if ($phase == 'install') {
+    // Check if openssl is enabled.
+    if (!extension_loaded('openssl')) {
```

Comment:
We need to file an issue for this.

The Encrypt API (in the encrypt module) basically can abstract the algorithm here. We may not need to leverage OpenSSL. Is there a specific reason?